buyer-api/src/queues/dataResponseQueue.js

```diff
@@ -83,21 +88,40 @@ export const processDataResponseJob = async (job) => {
   if (batchSize === -1) {
     logger.info(`addPrepareNotarizationJob will not be called on batchSize: ${batchSize}`);
   } else if (dataResponseIds.length >= batchSize) {
-    await clear(accumulatorId);
-    const batchId = await createBatch({ orderId, notaryAddress, dataResponseIds });
-    await addPrepareNotarizationJob({ batchId, price });
+    addSendNotarizationBatchJob({
+      accumulatorId,
+      orderId,
+      price,
+      notaryAddress,
+    });
   }
 
   const updateDataReseponse = { ...dataResponse, status: 'batched' };
   await dataResponses.store(dataResponseId, updateDataReseponse);
 
+  await dataResponsesLastAdded.store(accumulatorId, { notaryAddress, orderId, price });
+
   return updateDataReseponse;
 };
 
-queue.process(processDataResponseJob);
+export const sendNotarizationBatchJob = async (job) => {
+  const {
+    data: {
+      accumulatorId, orderId, price, notaryAddress,
+    },
+  } = job;
+
+  const dataResponseIds = await accumulator.safeFetch(accumulatorId);
+
+  await clear(accumulatorId);
```

Comment:
Careful with this part, I'm not sure if I got it right, but this `clear` call doesn't look good here:
* By splitting into these two jobs, the accumulator will be updated by the two of them.
* The above will cause problems in the batches we are creating

Reply:
Since is the same queue, the two jobs would not be launched at the same time